ui/src/types/api.ts

```diff
@@ -45,11 +45,12 @@ export interface Paper {
   title: string;
   authors: Author[];
   abstract: string | null;
-  url: string;
   venue: string | null;
   year: number | null;
   citationVelocity: number;
   influentialCitationCount: number;
+  primaryPaperLink: string;
+  paperLinkType: string;
```

Comment:
Should these be `string | null`? I think this depends on what the S2 API returns.

Reply:
upon inspecting a sample of papers, I found that some of the older papers (published in the 20th century) didn't get published on the web, therefore the api doesn't even have the `primaryPaperLink` field for it. (example: https://www.semanticscholar.org/api/1/paper/ac5e724f4223720c1b4f5bcc4c7b8c11ccea83c1)

will this cause the code to break, because the property doesn't exist?